examples/gno.land/r/matijamarjanovic/home/config.gno

```diff
@@ -0,0 +1,70 @@
+package home
```

Comment:
Curious, is this a `gno` thing, to have `config` package with this code, and duplicate it in `home`? Can it be a single thing? Are both needed?

Reply:
What I think was the reason for this is showing how to import realms from different packages but also dividing it into 2 logical sections. In the end I put the config realm in the same package as home.gno for easier deployment. Still I think it's better to leave them separate just for simplicity. There is no reason besides this to not put them in one file.

Reply:
I meant why is there a `home` package and `config` package, where the config/address handling code is duplicated. The config package can be removed, no?

Reply:
Yes, I understand now, I believe I fixed this in one of the newer commits.